### PR TITLE
chore: Update order of table metadata on TableDetailPage

### PR DESCRIPTION
**Pull request by ttannis**
Signed-off-by: Tamika Tannis <ttannis@lyft.com>

### Summary of Changes

New design calls for changing the order:
1. Last Updated, Date Range, & Tags on the left
2. Owners & Frequent Users on the right

### Tests

N/A no functionality has been added or changed.

### Documentation

N/A no functionality has been added or changed.

### CheckList

Make sure you have checked **all** steps below to ensure a timely review.

- [x] PR title addresses the issue accurately and concisely. Example: "Updates the version of Flask to v1.0.2"
  - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
- [x] PR includes a summary of changes, including screenshots of any UI changes.
- [x] PR adds unit tests, updates existing unit tests, **OR** documents why no test additions or modifications are needed.
- [x] In case of new functionality, my PR adds documentation that describes how to use it.
  - All the public python functions and the classes in the PR contain docstrings that explain what it does
- [x] PR passes all tests documented in the [developer guide](https://github.com/lyft/amundsenfrontendlibrary/blob/master/docs/developer_guide.md#testing)

### CODE REVIEW

**Review by dikshathakur3119 — APPROVED** [review]

File: amundsen_application/static/js/pages/TableDetailPage/index.tsx [hunk 1]
```diff
@@ -314,24 +308,30 @@ export class TableDetail extends React.Component<
...
                   )}
-                  <section className="metadata-section">
-                    <div className="section-title title-3">Frequent Users</div>
-                    <FrequentUsers readers={data.table_readers} />
-                  </section>
-                  {this.renderProgrammaticDesc(
-                    data.programmatic_descriptions.left
+                  {!data.is_view && (
+                    <section className="metadata-section">
+                      <div className="section-title title-3">Date Range</div>
```
Comment (Golodhros) [inline amundsen_application/static/js/pages/TableDetailPage/index.tsx:313]: Maybe a good chance to move strings to the constants file or updating the text styles?

**Review by Golodhros — APPROVED** [review]
LGTM

**Comment by feng-tao** [comment]
lgtm, is this coming from Knowl redesign?

**Comment by ttannis** [comment]
> lgtm, is this coming from Knowl's redesign?

Yes

**Review by Golodhros — APPROVED** [review]
LGTM

Thanks a lot for going the extra mile!

**Pull request merged**